.devcontainer/configure.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Configure script to setup VSCode DevContainer Environment."""
+
+from importlib.machinery import SourceFileLoader
+import inspect
+import json
+import os
+import shlex
+import subprocess
+import sys
+
+# pylint: disable=g-import-not-at-top
+try:
+  from shutil import which
+except ImportError:
+  from distutils.spawn import find_executable as which
+# pylint: enable=g-import-not-at-top
```

Comment:
shutil.which has been available since Python 3.3. No need for a conditional import

```suggestion
import shutil
import subprocess
import sys
```